modules/openapi-generator/src/test/resources/3_0/petstore.yaml

```diff
@@ -725,8 +725,23 @@ components:
             - available
             - pending
             - sold
+        family:
+          type: string
+      discriminator:
+        propertyName: family
       xml:
         name: Pet
+    Cat:
+      title: a Cat
+      description: A cat for sale in the pet store
+      type: object
+      required:
+        - color
+      allOf:
+        - $ref: '#/components/schemas/Pet'
+      properties:
+        color:
+          type: string
```

Comment:
Please remove this change since it's making too many unwanted changes. Also could you please point to test class(es) with Jackson? The one that we ask you to add

PS: thanks for your patience!

Reply:
The tests are in the two only *OpenApiGeneratorApplicationTests.java* (sorry I don't know how to link them) file of the PR.

As for the *petstore.yaml* modification, I can revert that but then I will have no class with inheritence in the openapi3 springboot sample. Do you want me to use another springboot (or a new) spring boot sample ?

PS: Also it look like that change show a bug in the Kotlin generator

Reply:
ahhh. in that case let's leave it without change. only spring-boot

Reply:
J'ai revert ce change 👍